test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,12 +35,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
 
+    @Test
+    public void isAnyNull_nullList_returnsFalse() throws Exception {
+        assertNoNull();
+        // no null objects
+        assertNoNull("abc", "ab", "a");
+        assertNoNull(1, 2);
+        assertNoNull(1, 1, "AB");
```

Comment:
Since these are already in a separate test case, should be omitted from this one, no?